EIPS/eip-5727.md

```diff
@@ -0,0 +1,596 @@
+---
+eip: 5727
+title: Semi-Fungible Soulbound Token
+description: An interface for soulbound tokens, also known as badges or account-bound tokens.
```

Comment:
Would be nice to somehow fit a description of "semi-fungible" in the description (alongside the soulbound description) to differentiate this proposal from the other soulbound tokens.